source/Octopus.Tentacle/Kubernetes/KubernetesConfig.cs

```diff
@@ -11,6 +11,7 @@ public static class KubernetesConfig
         public static string JobVolumeYaml => GetRequiredEnvVar($"{EnvVarPrefix}__JOBVOLUMEYAML", "Unable to determine Kubernetes Job volume yaml.");
         public static bool UseJobs => bool.TryParse(Environment.GetEnvironmentVariable($"{EnvVarPrefix}__USEJOBS"), out var useJobs) && useJobs;
         public static int JobTtlSeconds => int.TryParse(Environment.GetEnvironmentVariable($"{EnvVarPrefix}__JOBTTL"), out var jobTtl) ? jobTtl : 60; //Default 1min
+        public static int JobMonitorTimeoutSeconds => int.TryParse(Environment.GetEnvironmentVariable($"{EnvVarPrefix}__JOBMONITORTIMEOUT"), out var jobMonitorTimeout) ? jobMonitorTimeout : 1800; //30min
```

Comment:
This has me thinking ... we have deployment steps that run for hours (copying Azure files), we might need a longer timeout

Reply:
yeah. It's a fair concern I think when we are running as a worker. My gut feeling is let's unblock the k8s issue now and look at it more comprehensively when we get to this running as a worker.

Reply:
When we deploy a container (like Server), would it be sad if the upgrade migration took > 30min?

Reply:
It would depend if we had something like "is atomic" on, where the script waits until the pod is fully running